Core1Platform/BasicPhilosophy/04_ModuleOverview.md

```diff
@@ -0,0 +1,115 @@
+# Module overview
+
+In this part of the *Actindo Core1 Platform* documentation, you will find a brief overview on the most important *Actindo Core1 Platform* modules and technical components as well as a description of the data streams between these components.   
+See the interaction between the modules also in the overview graphic [Module overview &ndash; Diagram](./04a_GraphicModuleOverview.md).
+
+
+## Basic plugins and modules
+
+The following modules and plugins are the basis for the *Actindo Core1 Platform*. Their features control and process the data flow, the data processing, the data mapping as well as the communication with third party applications. In addition, it provides master data needed for a lot of modules.
+
+**Core1 Platform**   
+The *Core1 Platform* is underlying any plugin. It offers basic functions such as the following:
```

Comment:
"underlying any plugin"? All plugin and modules, außer Einstellungen/Settings...
Vielleicht sowas wie eine "underlying structure", auf die die Module und Plugins dann installiert werden und die Basisfunktionen zu Verfügung stellt?

Reply:
foundation